### PR TITLE
README: added note about GCCv6 problem

### PR DESCRIPTION
**Pull request by OlegHahm**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,6 +63,13 @@ RIOT is based on a microkernel architecture, and provides features including, bu
 * You want to start the RIOT? Just follow our [Getting started documentation](https://github.com/RIOT-OS/RIOT/wiki/Introduction)
 * The RIOT API itself can be built from the code using doxygen. The latest version is uploaded daily to http://riot-os.org/api.
 
+## KNOWN ISSUES
+* With latest GCC version (>= 6) platforms based on STM32 will raise some
```
Comment (miri64) [inline README.md]: There is also an issue on the `samd21` platform.

**Comment by OlegHahm** [comment]
addressed

**Comment by miri64** [comment]
ACK, please squash.

**Comment by OlegHahm** [comment]
squashed

**Pull request merged**